src/scss/footer.scss

```diff
@@ -0,0 +1,84 @@
+.footer {
+	color: #777;
+	padding: 10px 15px;
+	height: 20px;
+	text-align: center;
+	border-top: 1px solid #e6e6e6;
+}
+
+.footer:before {
```

Comment:
You are using scss, plz nest relevant tags together
https://sass-lang.com/guide
keyword: nesting